dist/elastio-snap-dkms-conf

```diff
@@ -4,6 +4,8 @@ PACKAGE_VERSION="@MODULE_VERSION@"
 # Items below here should not have to change with each driver version
 PACKAGE_NAME="elastio-snap"
 PRE_BUILD="genconfig.sh ${kernelver}"
+# This must be used in POST_INSTALL instead of in POST_BUILD, because DKMS seems to override the actual signature
```

Comment:
If DKMS signed was settupped we should not overwrite by our signature

Reply:
DKMS auto-signs it with a self-signed certificate. This certificate is not enrolled.